codecov.yml

```diff
@@ -4,6 +4,7 @@ ignore:
   - "benchmarking"
   - "api/benchmarking"
   - "primitives/benchmarking"
+  - "*_weight.go"
```

Comment:
I think there's no need to ignore them anymore as you now call them in `BaseWeight`